pom.xml

```diff
@@ -1291,6 +1315,7 @@
                         <reportsDirectory>${project.build.directory}/surefire-reports</reportsDirectory>
                         <junitxml>.</junitxml>
                         <filereports>TestSuite.txt</filereports>
+                        <argLine>-Xmx4g -Xss4m -XX:ReservedCodeCacheSize=512M</argLine>
```

Comment:
-Xmx4g -Xms4g -Xss32m -XX:ReservedCodeCacheSize=512M

Reply:
After changes, GA failed again, now shrink to `-Xmx2g -Xms2g -Xss8m -XX:ReservedCodeCacheSize=512M` and waiting GA results.